chanfitness/chanevent.go

```diff
@@ -0,0 +1,179 @@
+package chanfitness
+
+import (
+	"time"
+)
+
+type eventType int
+
+const (
```

Comment:
I've also been thinking about that, I think some kind of `stopMonitoring` event that needs to be added on shutdown. A bit annoying to have to record for each channel, so we could look at having some on-shutdown ChannelEventStore level event but that discussion can wait.

Reply:
That is actually a more general discussion that in a way also applies to this pr's scope. If there are multiple channels to the same peer, the peer activity is logged in each of those while there could also be one log per peer. Those logs would then be recombined when fitness data is requested. But I think we can start by storing it all on a per channel basis as you do now.

A `stopMonitoring` event may not be enough, because `lnd` can also be killed. When it then comes back up, there is no way to find out how long it was offline.